save-sandbox/src/main/kotlin/com/saveourtool/save/sandbox/entity/SandboxAgentStatus.kt

```diff
@@ -28,7 +28,7 @@ class SandboxAgentStatus(
     @Enumerated(EnumType.STRING)
     var state: AgentState,
 
-    @ManyToOne(fetch = FetchType.LAZY)
+    @ManyToOne(fetch = FetchType.EAGER)
```

Comment:
why eager here? I thought that LAZY will also ok here

Reply:
We already got similar problem: there is an issue with hibernate proxy object, which loaded lazy. Eager is the simpler solution